deploy/helm/rawfile-localpv/README.md

```diff
@@ -33,6 +33,7 @@ Kubernetes: `>= 1.21`
 | controller.tolerations | list | `[{"effect":"NoSchedule","key":"node-role.kubernetes.io/master","operator":"Equal","value":"true"}]` | Tolerations for controller component |
 | crds.csi.volumeSnapshots.enabled | bool | `true` | Install Volume Snapshot CRDs |
 | crds.enabled | bool | `true` | Disables the installation of all CRDs if set to false |
+| defaultFileSystem | string | `"ext4"` | Used when `fsType` is not set in the StorageClass |
```

Comment:
how was this being handled "today"?

Reply:
It is hardcoded to ext4 now
In the get_fs function that I have removed